keyboards/adelheid/info.json

```diff
@@ -0,0 +1,101 @@
+{
+    "keyboard_name": "Adelheid",
+    "url": "https://github.com/floookay/adelheid",
+    "maintainer": "floookay",
+    "width": 19.5,
+    "height": 6.5,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 82,
```

Comment:
```suggestion
```

Reply:
I added the key_count property because it was described as a required property in the docs here: https://docs.qmk.fm/#/reference_info_json?id=layout-format  
Is the documentation outdated? Or is this only relevant when you have multiple layouts with different number of keys (ANSI &#8596; ISO)

Reply:
Yeah, I think that page is outdated. The number of keys can already be deduced by counting the elements of the `layout` array.

Reply:
Right, should I update that section and open a PR so others don't follow suit?